business-registry-dashboard/app/pages/incorporateNow.vue

```diff
@@ -70,29 +70,37 @@ onMounted(async () => {
       }
     }
 
+    // Before the affiliation checks, add a guard clause
+    if (!business.nameRequest?.nrNumber) {
```

Comment:
Added a guard clause here just to fix an issue that typescript was complaining about.